### PR TITLE
Weakening range of causal relation between processes. This may be temporary. See https://github.com/oborel/obo-relations/issues/524

### LINKED ISSUE #524: relax range constraint on regulates
change from process to process or characteristic

(there are other tickets to be linked here but I can't find them)

This change is necessary for go-plus to be coherent (currently crytpic incoherence as not caught by elk 0.43)

alternative:

create three parallel relations and use these in GO for the RoBQ branch

The two alternatives are not mutually exclusive - I think it is most practical to first relax, then we can make the GO change. But this is actually quite urgent

### PR DESCRIPTION
**Pull request by cmungall**
Fixes #524

### CODE REVIEW

**Review by matentzn — APPROVED** [review]

File: src/ontology/ro-edit.owl [hunk 1]
```diff
@@ -4271,8 +4271,8 @@ AnnotationAssertion(obo:IAO_0000119 obo:RO_0002501 <http://purl.obolibrary.org/o
 AnnotationAssertion(obo:IAO_0000232 obo:RO_0002501 "Do not use this relation directly. It is intended as a grouping for a diverse set of relations, all involving cause and effect.")
 AnnotationAssertion(rdfs:label obo:RO_0002501 "causal relation between processes")
 SubObjectPropertyOf(obo:RO_0002501 obo:RO_0002410)
-ObjectPropertyDomain(obo:RO_0002501 obo:BFO_0000003)
-ObjectPropertyRange(obo:RO_0002501 obo:BFO_0000003)
+ObjectPropertyDomain(obo:RO_0002501 ObjectUnionOf(obo:BFO_0000015 obo:BFO_0000019))
```
Comment (matentzn) [inline src/ontology/ro-edit.owl:4274]: Its odd you go from BFO:3 to BFO:15 or BFO:19. Is that intended?

**Review by matentzn — CHANGES_REQUESTED** [review]
Just double check my comment

**Review by balhoff — CHANGES_REQUESTED** [review]
Just same question as Nico, otherwise looks good.

**Comment by cmungall** [comment]
occurent => process or quality.

yes this is intentional. Are there any useful siblings of process?

**Review by balhoff — APPROVED** [review]

**Review by balhoff — CHANGES_REQUESTED** [review]
If we're going to do this, we will need to do the same to 'precedes' and 'temporally related to', and 'preceded by', etc.

**Comment by balhoff** [comment]
Please hold off on merging this; I am going to make an alternative PR.

**Pull request closed without merging**